src/Jeffijoe.MessageFormat.MetadataGenerator/Plural/Parsing/LeftOperand.cs

```diff
@@ -0,0 +1,54 @@
+﻿namespace Jeffijoe.MessageFormat.MetadataGenerator.Plural.Parsing
+{
+    public interface ILeftOperand
```

Comment:
If you want, you can put these and the other AST nodes into their own files in a `AST` sub-directory

Reply:
Sounds good, done